csharp-advanced-topics/Dele_Types_Action_Func/Tests/Delegates_Test.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.2.7" />
+   
+    <PackageReference Include="coverlet.collector" Version="3.1.0" />
+	<PackageReference Include="MSTest.TestFramework" Version="2.2.8" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\Delegates_Action_Func\Delegates_Action_Func.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Delegates_Action_Func">
+      <HintPath>..\..\Delegates_Action_Func\bin\Debug\net6.0\Delegates_Action_Func.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.VisualStudio.TestPlatform.TestFramework">
+      <HintPath>..\..\Delegates_Action_Func\bin\Debug\net6.0\Microsoft.VisualStudio.TestPlatform.TestFramework.dll</HintPath>
+    </Reference>
+  </ItemGroup>

```

Comment:
This seems strange, what's happening here, why is it referencing DLLs from the output folder?

Reply:
I am not exactly sure why that reference is coming there. I have not added any extra references from outside. The only thing I had to do was to delete and add the test framework NuGet packages again since it did not take the older existing version. Maybe this reference was added after that. As I have mentioned in my emails before the PR,there was a problem with the NuGet manager for VS2022 which was resolved through multiple restores.